core/trino-main/src/main/java/io/trino/server/protocol/spooling/SpoolingConfig.java

```diff
@@ -122,7 +124,7 @@ public long getMaximumInlinedRows()
         return maximumInlinedRows;
     }
 
-    @Config("protocol.spooling.inlining.max-rows")
+    @Config("protocol.spooling.inlining.maximum-rows")
```

Comment:
are we using maximum elsewhere? Maybe you should adjust the other way around

Reply:
Ok, other way around it is